docs/source/cn/random.py

```diff
@@ -0,0 +1,35 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.bernoulli,
+    r"""
+    bernoulli(x, *, generator=None, out=None)
```

Comment:
```suggestion
    bernoulli(input, *, generator=None, out=None)
```
这里和英文版的参数不一致，应该是 input